dist/windows/installer.nsi

```diff
@@ -119,6 +119,12 @@ Section $(inst_startmenu) ;"Create Start Menu Shortcut"
 
 SectionEnd
 
+Section $(inst_startup) ;"Start qBittorrent on Windows start up"
+
+  WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Run" "qBittorrent" "$INSTDIR\qbittorrent.exe"
+
+SectionEnd
+
```

Comment:
This is wrong. But I don't blame you. This `Section` will run with admin rights and on the account the admin belongs to. Read the online docs for the nsis UAC plugin and/or `dist/windows/nsis plugins/UAC.zip/UAC.nsh`.
Basically the installer runs in 2 instances. One instance is run with user-level rights and the other with admin level rights. The scripts seen here are executed by the admin-level instance. You need an explicit command to execute something by the user-level instance.
Long story short it should look like this:
```nsis
Section $(inst_startup) ;"Start qBittorrent on Windows start up"

  !insertmacro UAC_AsUser_Call Function inst_startup_user ${UAC_SYNCREGISTERS}|${UAC_SYNCOUTDIR}|${UAC_SYNCINSTDIR}

SectionEnd

Function inst_startup_user

  WriteRegStr HKCU "Software\Microsoft\Windows\CurrentVersion\Run" "qBittorrent" "$INSTDIR\qbittorrent.exe"

FunctionEnd
```